requirements/quality.in

```diff
@@ -6,3 +6,4 @@ caniusepython3            # Additional Python 3 compatibility pylint checks
 isort                     # to standardize order of imports
 pycodestyle               # PEP 8 compliance validation
 pydocstyle                # PEP 257 compliance validation
+edx-lint
```

Comment:
Minor suggestion - could you put this in alphabetical order and include a comment on reason for inclusion like the other entries in the file?